continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -26,15 +27,14 @@ requirements:
   host:
     - python >=3.8
     - pip
-    - setuptools
```

Comment:
Should we add `dask-core` here as well (as was done [here]( https://github.com/conda-forge/distributed-feedstock/blob/c69dfd4b07dda15f51b6aa18bb8826a2d7e72341/recipe/meta.yaml#L26 ))?

Reply:
Wasn't sure if it was necessary for the nightly recipe since we are pinning the `dask-core` runtime dependency a little differently here; my assumption was that it was included in the host dependencies of the conda-forge recipe so we could pin the runtime dependency with `pin_compatible`.

If you think it's worth adding for sake of consistency, would something like this work:

```
  host:
    - python >=3.8
    - pip
    - dask-core >={{ dask_version }},<{{ next_release_version }}
```

Reply:
AIUI the main goal is to make sure that Conda recognizes the files associated with `dask-core` (and `distributed`) are already present (thanks to those packages being included). So Conda doesn't repackage those files in other dependent packages (creating file clobbering issues on install)

A separate question would be how do we handle the version constraint here. Agree that is likely different, which is ok (and am less worried about)

Reply:
The `dask-core` constraint is handled by `conda search`ing the latest nightly version  available from `dask/label/dev`, setting that as the minimum, and setting the max as the next patch release. For example, if we were to run it now, the constraint would look like:

```
    - dask-core >=2022.10.1a221026,<2022.10.1
```

IIRC, the blocker here was that `pin_compatible` wasn't able to pin that tightly on the minimum version, so we ended up having to do it manually; I can try again to verify.

On the other hand, `distributed` should be a lot easier to set a constraint for since we just use the same version / build as `dask`, so we should be able to get that working with `pin_exact`

Reply:
Cool well let's give it a try and if we run into issues we can always come back and fix it. Thanks Charles! 🙏